src/TRoomDB.cpp

```diff
@@ -252,21 +258,51 @@ TArea * TRoomDB::getArea( int id )
     }
 }
 
-void TRoomDB::setAreaName( int areaID, QString name )
+bool TRoomDB::setAreaName( int areaID, QString name )
 {
+    if( areaID < 1 ) {
+        qWarning( "TRoomDB::setAreaName((int)areaID, (QString)name): WARNING: Suspect areaID: %n supplied.", areaID );
```

Comment:
That's not how it works... time isn't a luxury, no need to introduce more maintenance burden. Unless we can get a compiler macro that inserts a signature, don't embed the function signature.